impeller/renderer/backend/gles/capabilities_gles.cc

```diff
@@ -158,6 +158,8 @@ bool CapabilitiesGLES::SupportsSSBO() const {
 }
 
 bool CapabilitiesGLES::SupportsTextureToTextureBlits() const {
+  // TODO(158388): switch this to true for improved performance
+  // on GLES 3.0+ devices.
```

Comment:
```suggestion
  // TODO(158388): Switch this to true for improved performance
  // on GLES 3.0+ devices. Note that this wasn't enabled because
  // there were some rendering issues on some devices.
```

Reply:
Also, I think this should print to a new issue.  Then we can close out the 2 related to the weird blit behavior (since it won't show up anymore after this).

Reply:
Filled https://github.com/flutter/flutter/issues/158523